templates/taxi/car_detail.html

```diff
@@ -12,7 +12,19 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>Drivers
+    {% if has_car %}
+      <form action="{% url 'taxi:add-or-remove-car' pk=car.id action='remove' %}" method="post" style="display:inline;">
+        {% csrf_token %}
+        <input type="submit" value="Delete me from this car" class="btn btn-danger" style="float:right">
+      </form>
```

Comment:
To implement this feature thorugh post method of CarDetailView you can keep one form with different inputs (yes/no) that you will later check in post method in your view. And you can include a hidden field here with car_id so you can pull it out in post method and use to create links between car and driver

Reply:
Good thinking - I overlooked this possibility and suggested a :crutch: